Mention order of concept exercises in config.json doc (#138)
exercism/docs: building/tracks/config-json.md

### Concept exercises

Each concept exercise is an entry in the `exercises.concept` array.

> The order of concept exercises will be the one displayed by the website so it should match the typical order in which they should be solved.

Reviewer: ```suggestion
Exercises are ordered on the website in the same order they are listed in this file, and should match the typical order in which they should be solved.
```
